docker/base/inject_opencv.Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG GENERIC_IMAGE
+
+########################## Install ROS2 Core ##########################
+FROM ${GENERIC_IMAGE} as wato_base
+
+RUN apt-get update
+RUN apt install -y libopencv-dev python3-opencv
+RUN export OpenCV_DIR=/usr/lib/x86_64-linux-gnu/cmake/opencv4/OpenCVConfig.cmake
```

Comment:
please do not touch base images. They should not be tampered with, especially when we are installing a very specific library like opencv. Just install this in the dependencies stage of the specific dockerfile you are building

Reply:
got it, removed now